gems/pending/spec/appliance_console/database_replication_standby_spec.rb

```diff
@@ -208,4 +215,85 @@
       expect(subject.start_postgres).to be_truthy
     end
   end
+
+  context "#configure_repmgrd" do
+    it "returns true if not setup to do the configuration" do
+      subject.run_repmgrd_configuration = false
+      expect(subject.configure_repmgrd).to be true
+    end
+  end
+
+  context "#write_repmgrd_config" do
+    let(:config_path) { Tempfile.new("repmgr_config").path }
+
+    before do
+      stub_const("ApplianceConsole::DatabaseReplication::REPMGR_CONFIG", config_path)
+    end
+
+    after do
+      FileUtils.rm_f(config_path)
+    end
+
+    it "appends the correct data to the config file" do
+      File.write(config_path, "some other stuff here\n")
+
+      subject.write_repmgrd_config
+
+      expect(File.read(config_path)).to eq(<<-EOS.gsub(/^\s+/, ""))
+        some other stuff here
+        failover=automatic
+        promote_command='repmgr standby promote'
+        follow_command='repmgr standby follow'
+        logfile=/var/log/repmgr/repmgrd.log
```

Comment:
Because it's a part of the replication configuration, it will only ever be called after the file has been truncated [here](https://github.com/ManageIQ/manageiq/blob/master/gems/pending/appliance_console/database_replication.rb#L52), but if we ever make it it's own separate option, then that would be an issue.

Do you think putting some more strict checking into `write_repmgrd_config` is worth it?